manifests/init.pp

```diff
@@ -36,6 +36,8 @@
 # $enable_ostree::        Boolean to enable ostree plugin. This requires existence of an ostree install.
 #                         type:boolean
 #
+# $github_username::      Github username to add remotes for
+#
```

Comment:
Granted, we could just have both -- the curated set and then the ability to do arbitrary.